pkg/project/common/local.env

```diff
@@ -0,0 +1,11 @@
+AEM_AUTHOR_USER=admin
```

Comment:
the purpose of that extra *.env file is to provide variables available/shared to both AEMC and Task tool